### ISSUE TITLE
Unclear output

### ISSUE THREAD

**Issue by jschwartzenberg** [zamaudio/intelmetool#4]
I'm not really sure what this output means:
```
$ sudo ./intelmetool
[sudo] password for julius: 
Good news, you have a `ICH9M LPC Interface Controller` so ME is present but can be disabled, continuing...
RCBA at 0xfed1c000
MEI not hidden on PCI, checking if visible
MEI device not found, huh?
exiting
```

Would it be possible to explain or maybe improve the output?

**Comment by zamaudio** [comment]
It is quite difficult to know what is going on with ME.  But from that output I am guessing you ran it on a X200 with libreboot installed.  Basically, it checks your hardware listing and looks up a table to see if you should be able to disable the ME or not, then it searches for it anyway, and returns an output of what it found.

**Comment by jschwartzenberg** [comment]
No, I do not have a libreboot system at all unfortunately. I just ran it on my Compal (Ahtec) notebook which I got (and am using since) end 2008, because I was interested to see what it would report.

So I still have no clue whether I've got this ME working or not now :)

**Comment by zamaudio** [comment]
I may not have the correct code written to unhide the pci device on all chipsets, especially the older ones.  So at this point, intelmetool cannot really tell you if you have a working ME on ich9m.
